raster-foundry/raster-foundry
# Remove Gatling load tests

## Overview

Now that Gatling load tests live in their own separate repo at https://github.com/raster-foundry/load-testing, remove load tests from the core repo.

### Checklist


### Notes

* This PR reverts a few older PRs that introduced and modified Gatling tests, including:
    * https://github.com/raster-foundry/raster-foundry/pull/4395
    * https://github.com/raster-foundry/raster-foundry/pull/4410
    * https://github.com/raster-foundry/raster-foundry/pull/4424
    * https://github.com/raster-foundry/raster-foundry/pull/4428

## Testing Instructions

 * Run `grep -R -F gatling .` and confirm that no references to Gatling remain

Connects https://github.com/azavea/raster-foundry-platform/issues/585
